src/lua/elrsV2.lua

```diff
@@ -801,23 +813,77 @@ local function handleDevicePageEvent(event)
       end
     end
   elseif edit then
-    if event == EVT_VIRTUAL_NEXT then
-      incrField(1)
-    elseif event == EVT_VIRTUAL_PREV then
-      incrField(-1)
+    if event ~= nil then
+      if event ~= 0 then
+	if touchState then -- Only touch events come with a touchState
+          if event == EVT_TOUCH_FIRST then
+            -- When the finger first hits the screen
+          elseif event == EVT_TOUCH_BREAK then
+            -- When the finger leaves the screen and did not slide on it
+          elseif event == EVT_TOUCH_TAP then
+            -- A short tap gives TAP instead of BREAK
+            -- touchState.tapCount shows number of taps
```

Comment:
Can remove these empty cases...

Reply:
Unused cases are removed in latest commit